roxie/ccd/ccdserver.cpp

```diff
@@ -1459,10 +1459,10 @@ class CRoxieServerActivity : implements CInterfaceOf<IRoxieServerActivity>, impl
             {
                 if (state==STATEstarted || state==STATEstarting)
                 {
+                    VStringBuffer err("STATE: activity %d reset without stop", activityId);
+                    ctx->queryCodeContext()->addWuException(err.str(), ROXIE_INTERNAL_ERROR, SeverityError, "roxie");
```

Comment:
This may show up in any query run "ad-hoc" but won't cause deployed queries run via XML to fail in any way. However it should give some ERROR: tracing in the log.